src/inx/ledger.rs

```diff
@@ -0,0 +1,216 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_block_stardust as bee;
+use packable::PackableExt;
+
+use super::InxError;
+use crate::{
+    maybe_missing,
+    types::{
+        context::{TryFromWithContext, TryIntoWithContext},
+        ledger::{ConflictReason, LedgerInclusionState, LedgerOutput, LedgerSpent},
+        tangle::MilestoneIndex,
+    },
+};
+
+#[allow(missing_docs)]
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct UnspentOutputMessage {
+    pub ledger_index: MilestoneIndex,
+    pub output: LedgerOutput,
+}
+
+#[allow(missing_docs)]
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Marker {
```

Comment:
should this not be `MarkerMessage` for consistency? :trollface:

Reply:
Yes